ui/app/components/pending-tx/confirm-send-ether.js

```diff
@@ -20,7 +20,7 @@ const {
   calcGasTotal,
   isBalanceSufficient,
 } = require('../send_/send.utils')
-const GasFeeDisplay = require('../send/gas-fee-display-v2')
+const GasFeeDisplay = require('../send_/send-content/send-gas-row/gas-fee-display/gas-fee-display.component')
```

Comment:
Needs to be `const GasFeeDisplay = require('../send_/send-content/send-gas-row/gas-fee-display/gas-fee-display.component').default`

Read here for explanation: https://stackoverflow.com/questions/33704714/cant-require-default-export-value-in-babel-6-x